NETLIFY.md

```diff
@@ -0,0 +1,30 @@
+# Build Status
+
+Once you "publish" your work on Forestry, you can view the status of your website building below. Once you see "Success", your updates have been published on your website. 
+
+## Status
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/492fe949-a5be-47bc-a482-aa4a42f7b3bf/deploy-status)](https://app.netlify.com/sites/csis-data-governance/deploys)
+
+## Status Guide
+
+Building = The update is in progress. Please do not publish again until this changes to one of the other statuses.  
+Success = The site has been updated. Your updates have now been published on your website.  
+Failed = The update did not publish. 
+
+## Troubleshooting tips
+
+Usually when a build fails, it's because content is improperly formatted.  To see what is causing the issue do the following.
+- Click on Site Activity (Circle next to the logo in the top left corner)  
+
+  ![Site Activity](site_activity.png)
+
+- If available, click on "Show Log" and review the error message provided by Forestry to determine the error.
```

Comment:
Add "You might need to scroll for a little while until the error is visible. The error message will be visible towards the end of the log."

Reply:
All set!